aws-observability-terraform/app-modules/variables.tf

```diff
@@ -167,5 +167,22 @@ variable "folder_installation_location" {
   default = "Personal Folder"
 }
 
+variable "sumologic_organization_id" {
```

Comment:
Please add comment "Org ID is required in App module for sharing App Folder with Sumo Organisation"

Reply:
Done